addons/amb_civ_population/fnc_civilianPopulationSystemInit.sqf

```diff
@@ -77,6 +79,8 @@ if(isServer) then {
     [ALIVE_civilianPopulationSystem, "ambientCrowdLimit", _ambientCrowdLimit] call ALIVE_fnc_civilianPopulationSystem;
     [ALIVE_civilianPopulationSystem, "ambientCrowdFaction", _ambientCrowdFaction] call ALIVE_fnc_civilianPopulationSystem;
     [ALIVE_civilianPopulationSystem, "enableInteraction", _enableInteraction] call ALIVE_fnc_civilianPopulationSystem;
+    [ALIVE_civilianPopulationSystem, "humanitarianHostilityChance", _humanitarianHostilityChance] call ALIVE_fnc_civilianPopulationSystem;
+    [ALIVE_civilianPopulationSystem, "maxAllowAid", _maxAllowAid] call ALIVE_fnc_civilianPopulationSystem;
```

Comment:
These calls to ALIVE_fnc_civilianPopulationSystem won't actually do anything since it doesn't handle those keys. If you want to use that as a getter/setter, you need to look below in the massive switch-case statement and add cases for "humanitarianHostilityChance" and "maxAllowAid" that handle those values appropriately.

Reply:
@DavisBrown723 Good pickup. That was meant to have been removed, will get it sorted.